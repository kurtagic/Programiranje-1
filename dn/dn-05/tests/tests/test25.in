58
0 33
2 38
0 24
0 35
0 49
0 69
1 29
1 17
2 50
1 49
0 26
1 28
0 39
2 54
0 42
1 55
0 26
2 30
2 76
0 23
1 67
2 35
1 70
1 19
2 20
0 37
1 15
2 62
2 68
2 48
2 30
2 66
2 62
1 69
2 53
1 65
2 56
1 49
1 15
2 41
1 55
2 24
1 52
2 40
0 68
2 20
2 55
2 54
2 66
1 22
0 27
2 30
2 67
1 38
2 75
2 66
2 57
2 48
