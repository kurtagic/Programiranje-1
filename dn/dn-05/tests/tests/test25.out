15: 8
17: 4
19: 4
20: 4
21: 4
22: 4
23: 11
24: 13
25: 13
26: 11
27: 11
28: 11
29: 11
30: 13
31: 13
33: 1
34: 1
35: 5
36: 5
37: 4
38: 8
39: 4
40: 8
41: 8
42: 6
43: 4
44: 4
45: 4
48: 13
49: 13
50: 3
51: 3
52: 5
53: 4
54: 14
55: 14
56: 12
57: 4
58: 4
62: 4
63: 4
65: 4
66: 12
67: 12
68: 10
69: 8
70: 8
71: 8
72: 1
75: 4
76: 4
77: 2
